python/ray/serve/scripts.py

```diff
@@ -263,6 +263,11 @@ def deploy(config_file_name: str, address: str):
         "will loop and log status until Ctrl-C'd, then clean up the app."
     ),
 )
+@click.option(
+    "--gradio",
+    is_flag=True,
+    help=("Whether to enable gradio visualization of deployment graph."),
```

Comment:
@simon-mo Do we to add a status like `EXPERIMENTAL`, `alpha`, or `beta` to the `help` message?